### PR TITLE
Only show default notes for facilitator applications. Add tests

### PR DESCRIPTION
**Pull request by aoby**

### CODE REVIEW

**Comment by aoby** [comment]
[P0 - Don't autopopulate notes in Teacher Applications](https://trello.com/c/0LRbITEi/314-p0-dont-autopopulate-notes-in-teacher-applications)

File: apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js [hunk 1]
```diff
@@ -1,47 +1,69 @@
...
+      form_data: {
+        firstName: 'First Name',
+        lastName: 'Last Name',
+        title: 'Title',
+        phone: 'Phone',
+        preferredFirstName: 'Preferred First Name',
+        accountEmail: 'accountEmail',
+        alternateEmail: 'alternateEmail',
+        program: 'program',
+        planOnTeachering: ['Yes'],
```
Comment (clareconstantine) [inline apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js:27]: is this supposed to be 'teachering'?

File: apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js [hunk 2]
```diff
@@ -1,47 +1,69 @@
...
+
+    it("Uses entered value for facilitator applications with notes", () => {
+      const facilitatorDetailView = mountDetailView('Facilitator', {notes: "actual notes"});
+      expect(facilitatorDetailView.state().notes).to.eql("actual notes");
+    });
+
+    it("Does not supply value for teacher applications with no notes", () => {
+      const facilitatorDetailView = mountDetailView('Teacher', {notes: ''});
+      expect(facilitatorDetailView.state().notes).to.eql('');
+    });
```
Comment (clareconstantine) [inline apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js:64]: should be `teacherDetailView`

File: apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js (hunk 1 above)
Comment (aoby) [reply apps/test/unit/code-studio/pd/application_dashboard/detail_view_contentsTest.js:27]: Ha, no but I like it :). This code was just moved - I didn't change it and didn't notice. Good catch. 

Actually this value, only present in facilitator applications, isn't specified in the props nor asserted in this test. I'm just going to remove it.

I accidentally merged, so will remove this in a followup...

**Pull request merged**